radix-engine/src/engine/call_frame.rs

```diff
@@ -261,29 +266,12 @@ impl<'a> LoadedSNodeState<'a> {
                 BorrowedSNodeState::Resource(resource_address, resource_manager) => {
                     track.return_borrowed_global_mut_value(resource_address, resource_manager);
                 }
-                BorrowedSNodeState::Bucket(_bucket_id, _bucket) => {}
-                BorrowedSNodeState::Proof(_proof_id, _proof) => {}
-                BorrowedSNodeState::Vault(vault_id, vault, value_type) => match value_type {
-                    ValueType::Owned => {
-                        owned_values.insert(
-                            StoredValueId::VaultId(vault_id.clone()),
-                            StoredValue::Vault(vault_id, vault),
-                        );
-                    }
-                    ValueType::Ref(ValueRefType::Uncommitted { root, ancestors }) => {
-                        let store = owned_values
-                            .get_mut(&StoredValueId::KeyValueStoreId(root))
-                            .map(|v| match v {
-                                StoredValue::KeyValueStore(_, store) => store,
-                                _ => panic!("Expected KV store"),
-                            })
-                            .unwrap();
-                        store.put_child_vault(&ancestors, vault_id, vault);
-                    }
+                BorrowedSNodeState::TrackedVault(vault_id, vault, value_type) => match value_type {
```

Comment:
`GlobalizedVault`?

Reply:
We reserve "globalize" for things that are accessible to all. Will keep it track for now